### DISCUSSION TITLE
Rad Pro: History marker

### DISCUSSION THREAD

**Discussion by alfmck** [Gissio/radpro#110] [Ideas]
I noticed that history graph, especially History24H for ambient measurement is less informative. Would be nice to have a moving marker and data in it. It could be difficult to do for 3 buttons devices, but for my 4-button EMF-390 it is used:

![IMG_3353](https://github.com/Gissio/radpro/assets/165254519/8118e6c7-ef74-428b-ad1a-d2c014cea490)

And one more:
I'm watching weekly data for my area in https://remap.jrc.ec.europa.eu/Advanced.aspx.

[Doc2.pdf](https://github.com/user-attachments/files/15937805/Doc2.pdf)

They supplied data for every hour. Surprising me that I see peaks, lasting for couple hours. The same peaks are registered in another stations 50-200 km away but shifted in time for couple of hours. The peaks are not big, but obviously informative. What do we think about it? Would it be advisable to have something common in radpro?

**Comment by gc01noob** [comment]
even a meaningful scale would be a huge improvement.

**Comment by DG0MG** [comment]
`The same peaks are registered in another stations 50-200 km away but shifted in time for couple of hours.`

This should be precipitation. The time shift is caused by the movement of the rain fronts. The rain washes radon out of the atmosphere, and the decay products cause a short-term increase in the dose rate on the ground around the measuring point.

The peak in the RadPro will have another cause, e.g. electromagnetic interference (was there a cell phone nearby?).

**Reply by alfmck** [reply]
To DG0MG: Really it is precipitation. Electromagnetic interference I haven't detected - when I place together Radpro device, EMF390 and cell phone in conversation state, Radpro device shows any reaction, while EMF390 shows 50 times increasing of RF and 35 times increasing of EMF.

I tried to register ambient data with "pancake" probe HP-265, connected to GC-01. This probe is 6 times more sensitive, than J321. Datalog set for 1 hour of getting data. Here are the results of Geigerlog and from https://remap.jrc.ec.europa.eu/Advanced.aspx.
![2024-10-30_103514](https://github.com/user-attachments/assets/fd764f32-ba46-47ef-8d9e-7ba7c1b75d60)
![2024-10-30_103515](https://github.com/user-attachments/assets/1ccb4261-3f72-476e-8346-1cc43f214a78)
(There is a bug in transferring timescale between Radpro and Geigerlog - sometimes data are incorrect with unknown reason).
It seems there is a good correlation between these graphs. Would be nice to have common sort of graphs in Radpro v 2.0.3

**Reply by DG0MG** [reply]
This looks surprisingly similar :)
